demos/python_demos/speech_recognition_demo/ctcdecode-numpy/CMakeLists.txt

```diff
@@ -34,6 +34,12 @@ endforeach()
 # SWIG-generated code causes some warnings; disable them.
 if(COMPILER_IS_GCC_LIKE)
     target_compile_options(${target_name} PRIVATE -Wno-narrowing)
+elseif(MSVC)
+    target_compile_options(${target_name} PRIVATE
+        /wd4244  # Disable conversion warning =
+        /wd4838  # Disable conversion warning in aggregate or list initialization
+        /wd4834  # Discarding return value of function with 'nodiscard' attribute
```

Comment:
I don't think this one should be disabled, because it's a real issue. :-) It's caused by an incorrect usage of `std::unique`, which should be fixed.

Reply:
It doesn't seem to be strong enough reason. It works after all … somehow :)

Reply:
It always works, until it doesn't. Maybe in our usage all words are already unique, but they might not be in the future. Why not fix it?